db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
I guess it should be NULL for db as well - with blank you allow this field to be empty for ORM but for database it actually expects some value therefor it will trigger an error

Reply:
I changed it, but in README we have: 'description - a text field, can be blank'. That`s a reason why I wrote only 'blank'